src/Blazored.Video/BlazoredVideo.razor.Properties.cs

```diff
@@ -0,0 +1,215 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+using Blazored.Video.Support;
+using Microsoft.JSInterop;
+
+namespace Blazored.Video
+{
+	public partial class BlazoredVideo
+	{
+		public bool Autoplay
+		{
+			get { return GetValue<bool>(); }
+			set { SetValue(value); }
+		}
+
+		public object AudioTracks
```

Comment:
I'm not sure it is wise to add properties that will just throw an exception - I think it would be a better experience to just no provide these - was there a reason in your mind for having them ?

Reply:
I thought about that like you in the first place but if the properties are there, and some day they are supported you could use them without recompiling the lib.
( object because of lazyness )